src/app/gallery/[type]/page.js

```diff
@@ -8,15 +10,24 @@ export async function generateStaticParams() {
 
 const Page = ({ params }) => {
   const PAGES = {
-    fall: "Fall",
-    winter: "Winter",
-    spring: "Spring",
+    fall: "Fall 2023",
+    winter: "Winter 2024",
+    spring: "Spring 2024",
     past: "Past",
   };
   if (PAGES.hasOwnProperty(params.type)) {
     return (
-      <div className="text-7xl">
+      <div className="flex flex-col text-7xl items-center pt-10">
         <Title text={PAGES[params.type]} food={"chip"} />
+        <div className="grid grid-cols-3 w-4/5 py-10 gap-2">
+          {PICTURES[params.type].map((image, index) => (
```

Comment:
it will throw a build error since there's a chance PICTURES[params.type] might not exist.

use an && operator